docs/user/authentication_and_authorization.md

```diff
@@ -13,155 +13,165 @@ These operations respect the rules of the section 6.3.5 of the NGSI-LD specifica
-        "isDeleted": {
-            "type": "Property",
-            "value": true
-        },
-        "right": { 
-            "type": "Property", 
-            "value": "canAdmin" 
-        }, 
-        "specificAccessPolicy": { 
-            "type": "Property", 
-            "value": "AUTH_READ" 
-        }, 
-        "canRead": [  
-            {  
-                "type": "Relationship",  
-                "object": "urn:ngsi-ld:User:2194588E-D3CE-47F9-B060-B77DB6EAAAD8",
-                "datasetId": "urn:ngsi-ld:Dataset:2194588E-D3CE-47F9-B060-B77DB6EAAAD8",
-                "subjectInfo": {
-                    "type": "Property",
-                    "value": {
-                        "kind": "User",
-                        "username": "stellio-user"
-                    }
-                }
-            }, 
-            {  
-                "type": "Relationship",
-                "object": "urn:ngsi-ld:Client:D7A09461-4FD1-4B96-A15E-1DCABD11FE04",
-                "datasetId": "urn:ngsi-ld:Dataset:D7A09461-4FD1-4B96-A15E-1DCABD11FE04",
-                "subjectInfo": {
-                    "type": "Property",
-                    "value": {
-                        "kind": "Client",
-                        "clientId": "client-id"
-                    }
-                }
-            }, 
-            {  
-                "type": "Relationship",  
-                "object": "urn:ngsi-ld:Group:5AD29EF5-5427-46DA-9573-7CA03F842701",
-                "datasetId": "urn:ngsi-ld:Dataset:5AD29EF5-5427-46DA-9573-7CA03F842701",
-                "subjectInfo": {
-                    "type": "Property",
-                    "value": {
-                        "kind": "Group",
-                        "name": "Stellio Team"
-                    }
-                }
-            } 
-        ], 
-        "canWrite": [ 
-            … 
-        ], 
-        "canAdmin": [ 
-            … 
-        ],
-        "isOwner": {
-            // only if user is not the owner of the entity, in which case the value of the "right" property will be "isOwner"
-        }
-        "@context": "https://easy-global-market.github.io/ngsild-api-data-models/authorization/jsonld-contexts/authorization-compound.jsonld"
-    },
-    {
-        ...
-    }
-]
+-  GET /auth/permissions/{id}
+
+ ```json
+{
+  "id" : "urn:permission:my:id",
+  "type" : "Permission",
+  "target" : {
+    "id" : "my:entity:id"
+  },
+  "assignee" : "subjectID", 
+  "action":  "write",
+  "@context": "https://easy-global-market.github.io/ngsild-api-data-models/authorization/jsonld-contexts/authorization-compound.jsonld"
+}
 ```
 
-The body also contains the other users who have a right on the entity.
+You can ask to retrieve the entity and the assignee information in the same request by specifying
+- details=true
+
+The result will look like this :
+````json
+{
+  "action" : "read",
+  "assignee" : {
+    "kind" : "Group",
+    "name" : "Stellio Team"
+  }, 
+  "assigner" : {
```

Comment:
the assigner cannot be a group since a group is only a logical grouping of users (and you can't authenticate as a group)